core/settings.py

```diff
@@ -285,7 +285,13 @@
 PELICAN_FRONTEND_URL = os.getenv("PELICAN_FRONTEND_URL")
 
 # WARNING: If you change the production default, update `Dockerfile_django` and `docker-compose.yaml` to match.
-EXPORTER_DIR = os.getenv("EXPORTER_DIR", "/data/exporter" if production else BASE_DIR / "data" / "exporter")
+EXPORTER_MOUNT_DESTINATION = os.getenv(
+    "EXPORTER_MOUNT_DESTINATION", "/data/exporter" if production else BASE_DIR / "data" / "exporter"
+)
+# WARNING: The production default needs to sync with Spoonbill's configuration.
```

Comment:
Not true - the default can be anything. The deploy repo is responsible for setting this environment variable so that Spoonbill receives the correct value.

```suggestion
```

Reply:
Ah so I need to update the deploy repo too. I was thinking that we could set this as the default and not change the deploy repo, but what you suggest makes more sense.